pkg/server/credtemplate/builder_test.go

```diff
@@ -386,10 +386,20 @@ func TestBuildDownstreamX509CATemplate(t *testing.T) {
 			},
 		},
 		{
-			desc: "override X509SVIDTTL",
+			desc: "default legacy TTL",
```

Comment:
can you add a unit test to see what happens if UseLegacyDownstreamX509CATTL == false, and ttl == 0?

Reply:
This test case already exists (see the first test case, "defaults")